html/captive-portal/lib/captiveportal/PacketFence/DynamicRouting/Module/Root.pm

```diff
@@ -244,6 +244,11 @@ sub apply_new_node_info {
     my ($self) = @_;
     get_logger->debug(sub { use Data::Dumper; "Applying new node_info to user ".Dumper($self->new_node_info)});
 
+    if(pf::node::is_max_reg_nodes_reached($self->current_mac, $self->new_node_info->{pid}, $self->new_node_info->{category})) {
+        $self->render("max-nodes-reached.html", { show_manage_devices => $self->app->profile->showManageDevicesOnMaxNodes });
```

Comment:
It looks that max-nodes-reached.html is missing.

Reply:
fixed